docs/cf-cli-plugin.md

```diff
@@ -0,0 +1,173 @@
+# Application Autoscaler Plugin for the Cloud Foundry CLI
+The Application Autoscaler offers a plugin for the Cloud Foundry Command Line Interface CLI. The plugin enables users to:
+  * attach or detach scaling policies to apps
+  * view attached scaling policies and view the scaling histories
+  * view the metrics sent by the app
```

Comment:
I think there is no context here specifying which app is meant by “the app”. I would be less specific here. As an example:
```suggestion
  * view the metrics sent by apps
```